validator/keymanager/remote-web3signer/v1/web3signer_types.go

```diff
@@ -0,0 +1,326 @@
+package v1
+
+/* Web3Signer Specs are found on the following link: https://consensys.github.io/web3signer/web3signer-eth2.html */
```

Comment:
Links can go stale, or become broke in the future. Recommend to just ask the user to look up Consensys' Web3Signer API specification and that these types represent version 1.0